docs/js.md

```diff
@@ -0,0 +1,162 @@
+### [Primative: Null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null)
+
+* _symbol_: `null`
+* _pronunciation_: "null", "null value", "is null"
+* _examples_:
+```javascript
+null // how exciting
+```
+
+### [Primative: Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)
+
+*_symbol_: 'true' 'false'
+*_pronunciation_"boo-lee-an"
+*_examples_:
+```javascript
+50 < 100 == true
+50 > 100 == false
+
+('meh' != meh) == true
+```
+
+### [Primative: Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)
+
+*_symbol_: 'The numeric value of the object being created.'
+*_pronunciation_"number" "integer"
+*_examples_:
+```javascript
+1
+2
+3
+10
+```
+
+### [Primative: String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)
+
+*_symbol_: 'a sequence of characters'
+*_pronunciation_"string" "123456789" "abcdefghijk"
+*_examples_:
+```javascript
+'I like big cats' + '\n and I cannot lie'
+```
+
+### [Built in Values: Infinity](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+
+*_symbol_: 'Infinity'
+*_pronunciation_: "in-fin-it-ee"
+*_examples_:
+```javascript
+3 * infinity == infinity
+```
+
+### [Built in Values: NaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+
+*_symbol_: 'NaN'
+*_pronunciation_: "not a number", "nahn"
+*_examples_:
+```javascript
+"cat" == NaN
+```
```

Comment:
So, I'm not sure this comparison holds true... Be sure to run your examples through the REPL or `node` command to confirm! You don't want to document a _bad_ example.